src/makeCalculator.js

```diff
@@ -37,7 +37,41 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    add(num) {
+      this.result += num;
+
+      return this.result;
+    },
+    subtract(num) {
```

Comment:
```suggestion
    add(num) {
      this.result += num;

      return this.result;
    },
    
    subtract(num) {
```
Add empty lines between methods for better readability